src/server/protocol.ts

```diff
@@ -3478,6 +3478,8 @@ namespace ts.server.protocol {
         readonly includeInlayFunctionLikeReturnTypeHints?: boolean;
         readonly includeInlayEnumMemberValueHints?: boolean;
         readonly autoImportFileExcludePatterns?: string[];
+
+        readonly disableLineTextInReferences?: boolean;

```

Comment:
I don't think we have a firm convention, but I think we usually use positive names (includeX) and then (potentially) treat `undefined` as the default value, rather than fals-y.  @DanielRosenwasser or @andrewbranch might have thoughts?  It's a little hard to find a precedent because we usually opt in to doing, rather than not doing, things.

Reply:
```ts
includeLineTextInReferences
```
?

Reply:
That sounds good to me, but I'd really like to hear from someone else so it's not just my intuition driving the change.

Reply:
I think your intuition is right, except for the fact that dropping something information is out of the norm. I can go either way on this.

Reply:
@mjbvz seems happy and I don't expect anyone else to ever set it. 😆 

I'm fine with leaving it.  Thanks for your patience, @a-tarasyuk.